react-app/src/components/ProposalDetailScreen/ProposalDetailScreenAPI.ts

```diff
@@ -405,10 +414,12 @@ export function useProposalQuery(): {
             : null,
           submitTime: new Date(proposal.submitTime),
           turnout: proposal.turnout ?? null,
-          remainingVotingDays:
+          remainingVotingDuration:
             proposal.votingStartTime &&
             proposal.votingEndTime &&
-            differenceInDays(new Date(proposal.votingEndTime), Date.now()),
+            formatDistance(new Date(proposal.votingEndTime), new Date(), {
+              locale: locale === Locale.zh ? zhTW : undefined,
+            }),
```

Comment:
Let's create a map in LocaleProvider for this, as we don't know whether there will be more languages in the future